tests/controllers/sib-deletion/chainsaw-test.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: sib-deletion
+spec:
+  timeouts:
+    assert: 10s
+  steps:
+
+  - name: step-01 
+    try:
+      - apply:
+          # file is relative to the test folder
+          file: ../../../examples/namespaced/dns-manipulation-si-sib.yaml
+      - assert:
+          file: ../../../examples/namespaced/dns-manipulation-si-sib.yaml
+  - name: step-02
+    try:
+      - script:
+          content: kubectl delete sib -n $NAMESPACE dns-manipulation-binding
```

Comment:
Leverage the full capabilities of chainsaw. In this instance, chainsaw's [delete](https://kyverno.github.io/chainsaw/latest/operations/delete/#usage-in-test) operation to perform the necessary deletion.